openscenario/openscenario_interpreter/src/syntax/position.cpp

```diff
@@ -26,7 +26,7 @@ Position::Position(const pugi::xml_node & node, Scope & scope)
     choice(node,
       std::make_pair(         "WorldPosition", [&](auto && node) { return make<        WorldPosition>(node, scope); }),
       std::make_pair( "RelativeWorldPosition", [&](auto && node) { return make<RelativeWorldPosition>(node, scope); }),
-      std::make_pair("RelativeObjectPosition", [&](auto && node) { throw UNSUPPORTED_ELEMENT_SPECIFIED(node.name()); return unspecified; }),
+      std::make_pair("RelativeObjectPosition", [&](auto && node) { return make<RelativeObjectPosition>(node, scope); }),
```

Comment:
Adjust the column alignment according to the style of the original code. Do not try to break the style to reduce diffs.

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/1030/commits/7824f61848f356476a46d9e9555c9b820df1d519